CMSgov/design-system
# [WNMGDS-897] Add analytics guidelines tab

## Summary
Add an analytics section under the Guidance dropdown on the document site

### Added
Added `analytics` nav under the guidance section

### Changed
Changed Alert component to track events for `error`, `warn` and `success` variations only

## How to test
Analytics guidelines section:
- Go to the demo url: http://design-system-demo.s3-website-us-east-1.amazonaws.com/WNMGDS-897/add-analytics-guidelines-tab/guidelines/analytics/
- Check the analytics guidelines

Alert Component:
- Clone the branch and test locally
- Update the Alert React examples to include the regular information alert and one for `error`, `warn` and `success` variations. 
- Go to the Alert React examples and inspect the network tab (or use omnibug) to check that analytics events are tracked for the  `error`, `warn` and `success` variations only.
